src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,86 @@
+package duke;
+
+import duke.Deadline;
+import duke.Event;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Storage {
+    Path currentRelativePath = Paths.get("");
+    String s = currentRelativePath.toAbsolutePath().toString();
+
+    public Storage() {
+
+    }
+
+    public ArrayList<Task> load() throws DukeException {
```

Comment:
Since your save method is named `saveTasks`, would it be better to name this `loadTasks`? For standardisation

Reply:
Yes, you are right.